client/src/test/java/com/sinch/sdk/domains/verification/adapters/VerificationStatusServiceTest.java

```diff
@@ -45,42 +45,39 @@ public void initMocks() {
   void getByIdentity() throws ApiException {
 
     when(api.verificationStatusByIdentity(eq("endpoint string"), eq("sms")))
-        .thenReturn(VerificationStatusDtoTest.expectedVerificationCalloutDto);
+        .thenReturn(VerificationStatusResponseInternalTest.expectedVerificationPhoneCallDto);
```

Comment:
Can you double-check you have pushed the right version of the files? I think we already spoke about the mismatch sms / phone call.

Reply:
Yes, my bad.
Fixed